### PR TITLE
Soen18 patient quarantine

### PR DESCRIPTION
**Pull request by stevenvuong00**
# Description
Worked with @NicholasYip on this small ticket.  All the information was taken from the government.

Implements/fixes [SOEN-#](URL to Jira ticket)

- new implementation
- added a new route to get the required information about the patient

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] Tested component rendering for the frontend
- [X] Test backend to know if the patient has covid or not

# Screenshots
image

### CODE REVIEW

File: backend/controllers/patient-controller.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const RequiredDetails = require("../models/required-details");
 const Patient = require("../models/patient");
 
 async function getRequiredDetails(req, res) {
-    console.log(req.params.accountId );
+    console.log(req.params.accountId);
```
Comment (jfv-2000) [inline backend/controllers/patient-controller.js]: Remove unnecessary logs if possible

**Review by jfv-2000 — APPROVED** [review]
Great work 💯 , just remove an unnecessary console log to keep it mr.clean 🧽

**Review by amqkv — APPROVED** [review]
Good work!

**Review by KelvinTrang — APPROVED** [review]
Good work bois!

**Pull request merged**